kolibri/plugins/facility/assets/src/views/FacilityIndex.vue

```diff
@@ -112,6 +114,12 @@
         return false;
       },
     },
+    beforeMount() {
```

Comment:
I think this is redundant with the URL changes you've made, so I'd rather just get rid of the change in this file.

Plus `isSubsetOfUsersDevice` is not defined for the plugin data for _this_ plugin.

Reply:
Thanks for the keen eye here! I reverted the CoachIndex changes but missed this one.